docs/.vuepress/components/VxpSideDrawerDoc.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+	<StackLayout>
+		<VxpSideDrawer
+			id="VxpSideDrawer"
+			ref="drawer"
+			class="sidedrawer-doc-container">
+			<FlexboxLayout
```

Comment:
use editor config to avoid indentation problems

Reply:
I have the extension installed actually but maybe forgot to format file.